database/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM postgres:latest
+
+ENV POSTGRES_USER=user
+ENV POSTGRES_PASSWORD=password
```

Comment:
💀 We should agree on a password, and have it in a local file where a gitignore prohibits us from committing it.